src/components/Header.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useContext, useEffect, useRef, useState } from 'react';
+import classNames from 'classnames';
+import { TodoContext } from '../context/TodoContext';
+import { Error } from '../types/ErrorMessage';
+import * as TodoService from '../api/todos';
```

Comment:
```suggestion
import * as todoService from '../api/todos';
```
it's not a class, type or react component